deepcell/applications/cell_tracking.py

```diff
@@ -43,11 +42,41 @@
               'saved-models/NuclearTrackingInf-7.tar.gz')
 MODEL_HASH = '46c5cbe0e362ed4ffdb54e11e3dbdc3e'
 
-
 ENCODER_PATH = ('https://deepcell-data.s3-us-west-1.amazonaws.com/'
                 'saved-models/NuclearTrackingNE-7.tar.gz')
 ENCODER_HASH = '7c30b39b2d2db0519f57d97194195504'
 
+MODEL_METADATA = {
+    'batch_size': 8,
+    'n_layers': 1,
+    'graph_layer': 'gat',
+    'epochs': 50,
+    'steps_per_epoch': 1000,
+    'validation_steps': 200,
+    'rotation_range': 180,
+    'translation_range': 512,
+    'buffer_size': 128,
+    'n_filters': 64,
+    'embedding_dim': 64,
+    'encoder_dim': 64,
+    'lr': .001,
+    'data_fraction': 1,
+    'norm_layer': 'batch',
+    'appearance_dim': 32,
+    'distance_threshold': 64,
+    'crop_mode': 'resize',
+}
+
+DISTANCE_THRESHOLD = 64
+APPEARANCE_DIM = 32
+CROP_MODE = 'resize'
```

Comment:
Just double-checking that these params are intended to appear *both* in the `MODEL_METADATA` dict and here!